src/com/itschool/session4/homework/binaryarithmeticoperators/ex1.java

```diff
@@ -0,0 +1,38 @@
+package com.itschool.session4.homework.binaryarithmeticoperators;
+
+import java.util.Scanner;
+
+/*Write a Java program to take two numbers as input and display the result of addition,
+subtraction, multiplication, division, and modulus.
+ */
+
+
+public class ex1 {

```

Comment:
Rename the class to match the Java convention (class name should start with upper-case letter).